package.json

```diff
@@ -20,23 +20,20 @@
   "license": "BSD-3-Clause",
   "dependencies": {
     "@babel/eslint-parser": "7.19.1",
-    "@graphql-tools/merge": "8.3.6",
-    "@graphql-tools/schema": "9.0.4",
-    "@graphql-tools/utils": "8.12.0",
-    "@graphql-yoga/node": "2.6.0",
+    "@graphql-tools/schema": "9.0.12",
+    "@graphql-tools/utils": "9.1.3",
     "@parse/fs-files-adapter": "1.2.2",
     "@parse/push-adapter": "4.1.2",
+    "@whatwg-node/fetch": "0.5.3",
     "bcryptjs": "2.4.3",
-    "body-parser": "1.20.1",
```

Comment:
This is deprecated in favor of `express.X` methods.

Reply:
And this isn't used anywhere else in the codebase?

Reply:
Except the places I changed, no.

Reply:
This should be done in a separate PR. This PR should only touch the yoga related files in the source code.

Reply:
Ok I reverted my changes.